
<launch>
  <arg name = "interface" default = "dev/ttyS0"/>
  <arg name = "baudrate" default = "115200"/>
  <arg name = "rate" default = "800"/>
  <arg name = "mode" default = "binary"/>/>


  <node name ="imu" pkg="imu_manager" type="imu_manager_node" output="screen">
  <param name = "interface" value ="$(arg interface)"/>
  <param name = "baudrate" value = "$(arg baudrate)"/>
  <param name = "rate" value = "$(arg rate)" />
  <param name = "mode" value = "$(arg mode)" />

  <param name = "imu_topic" value = "imu" />
  <param name = "frame_id" value = "vectorNav" />
  <param name = "reload_service_name" value = "reload" />
  <param name = "stop_service_name" value = "abort" />
  </node>


</launch>






